sh.xaviers.Opentapes.yml

```diff
@@ -0,0 +1,48 @@
+app-id: sh.xaviers.Opentapes
+command: opentapes
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: opentapes
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node16/bin:/usr/lib/sdk/rust-stable/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    CARGO_HOME: /run/build/Opentapes/cargo
+    RUST_BACKTRACE: '1'
+cleanup:
+  - /node_modules
+  - /dist
+  - /src-tauri/target
+modules:
+  - name: Opentapes
+    sources:
+      - type: archive
+        # For next release: https://codeberg.org/xaviers/Opentapes/archive/0.x.0.tar.gz
+        url: https://codeberg.org/xaviers/Opentapes/archive/main.tar.gz
```

Comment:
Note this URL is not stable and should be changed to point to specific commit instead of branch name.